.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Setup cmake
       uses: jwlawson/actions-setup-cmake@v1.12
       with:
-        cmake-version: '3.13.x'
+        cmake-version: '3.17.x'
```

Comment:
Should we change `cmake_minimum_required(VERSION 3.13)` in CMakeLists.txt to `cmake_minimum_required(VERSION 3.17)`?

If not, we should add a comment here to explain why the cmake version here differs from the minimum required cmake version in CMakeLists.txt.